alpine_test/Dockerfile

```diff
@@ -23,6 +23,6 @@ RUN apk update && \
     rm -rf /var/cache/apk/*
 
 COPY init_snap /usr/local/bin/init_snap
-COPY snapd.conf /etc/snap/snapd.conf
+COPY snapteld.conf /etc/snap/snapteld.conf
```

Comment:
Add a symlink from /etc/snap/snapd.conf -> /etc/snap/snapteld.conf otherwise if we get an old binary it will just fail.